dist/source/repo-creator/repository-chooser.html

```diff
@@ -28,17 +28,29 @@
 					</ul>
 				</div>
 			</div>
+            <div class="app-search-error">
+                    <div class="message">
+                        <div class="text bg-danger">
+                            <i class="fa fa-warning"></i> Oops something went wrong! Please try after some time.
+                        </div>
+                    </div>
+                </div>
```

Comment:
In .editorconfig, the ident style is set to "2 space" not tab. I can change my editor to it also.